learning_assistant/api.py

```diff
@@ -103,7 +103,7 @@ def get_block_content(request, user_id, course_id, unit_usage_key):
     return cache_data['content_length'], cache_data['content_items']
 
 
-def render_prompt_template(request, user_id, course_run_id, unit_usage_key, course_id):
+def render_prompt_template(request, user_id, course_run_id, unit_usage_key, course_id, template_string):
```

Comment:
Questioin: is the template_string required? Either way, would we need to handle error cases?

Reply:
yeah it has no default so you'd need to pass this. IMO I don't think we need to null check each param here, the linter already covers us since won't let you invoke a function w/ missing arguments